src/components/WaiveForm.tsx

```diff
@@ -83,6 +83,8 @@ const WaiveForm: React.FC<{}> = () => {
     const invalidText =
         'Reason must have detailed explanation. Provide links to issues, bugs, etc';
     const helperText = '';
+    const agreementText =
+        'I am aware about the impact of this action, and I take full responsibility if something happens. E.g., (broken builds may affect the release of RHEL), etc.';
```

Comment:
Let's try to change this.

Ideas:
`agreementText = 'I agree and acknowledge the above information'`

Add above a text-widget with next contents:

`Waiving test results may have impact info RHEL release process. Broken builds can lead to broken RHEL composes. Unverified or failed builds can cause issues in system integration. 
Before waiving this tests it is good to check other possible options, in particular
some CI-systems can fail due to oputages and different circumstances. It is good to restart test or to contact CI-owners for assistance. Prodceed waiving test-result only when other efforts have not succeeded.`

@jimbair @dcantrell Preston please help review the above waiving message, is there way to rewrite it in more good English?

Reply:
Here are my edits:
Waiving test results may have an impact on the RHEL release process. Broken builds can lead to broken RHEL composes and unverified or failed builds can cause issues in system integration. Before waiving these tests it is good to check other possible options, in particular some CI-systems can fail due to outages and different circumstances. It is good to restart the test or to contact CI-owners for assistance. Proceed waiving test-result only when other efforts have not succeeded.

Reply:
There are some options. I prefer the first one. What about you ?


![image](https://user-images.githubusercontent.com/26419281/192955620-73c376d2-6b53-4a01-b461-ec79301ea075.png)

![image](https://user-images.githubusercontent.com/26419281/192956135-984a9c0f-53c9-4de0-8c73-2b1a33dc43c1.png)

![image](https://user-images.githubusercontent.com/26419281/192956188-411521a1-3b97-468e-9513-f50e737c61f0.png)

Reply:
Hi, I do not have strict preference here. I like both: 2 & 3.

Reply:
I think I also prefer number 2 among these.

A quick idea: Have you tried placing the disclaimer at the top? I quite like the journey: read disclaimer → realise possible impacts → write appropriate waiver → confirm by checking checkbox, implicitly recall disclaimer → double confirm by clicking _Submit_.

![waive-form-mock](https://user-images.githubusercontent.com/134321/193036535-c03390d6-bbf7-495f-9ab3-5cac20b544a8.png)

Reply:
Yeah, good points.

Reply:
I like 2 and 3 (maybe 3 more, because it makes the text more visible?). What I do not like on Matej's suggestion is that "I agree and acknowledge the above information" checkbox refers to the text which is above the "Reason". It seems to me like the "Reason" and this input box does not logically belong there.

Reply:
![image](https://user-images.githubusercontent.com/26419281/194025112-343bd3c1-a203-4539-8e86-ee9bce8daa2b.png)

I didn't find the option how to add attribute required only to the checkbox without label, so that's the final result, I think. I will wait until it's approved and then push it :+1:

Reply:
Looks good to me. Regarding the checkbox, you should be able to use the [`isRequired` attribute](https://www.patternfly.org/v4/components/checkbox/#required-input).